### ISSUE TITLE
[bug] `conan_build_info --v2 create` is unable to find item

### ISSUE THREAD

**Issue by GordonJess** [conan-io/conan#6073]
Receive error message:
`ERROR: 404: Unable to find item`
when trying to create build info using the `conan_build_info --v2 create` command  

### Environment Details (include every applicable attribute)
  * Operating System+version: Windows 10
  * Compiler+version: n/a
  * Conan version: 1.20.3
  * Python version: Python 3.7.3

### Steps to reproduce (Include if Applicable)
```
conan install
conan build
conan package --package-folder=Package
conan export-pkg --force --ignore-dirty --package-folder=Package --lockfile conan.lock
conan upload -r <repo> --all <ref>
conan_build_info --v2 create buildinfo.json --lockfile conan.lock --user <user> --password <pass>
```

### Logs (Executed commands with output) (Include/Attach if Applicable)
tracefile contents:
```
{"_action": "COMMAND", "name": "install", "parameters": {"build": null, "channel": null, "env": null, "generators": null, "install_folder": null, "lockfile": null, "manifests": null, "manifests_interactive": null, "name": null, "no_imports": true, "options": null, "path": "..", "profile_names": null, "remote_name": null, "settings": null, "update": false, "user": null, "verify": null, "version": null}, "time": 1573724181.3800154}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "PyRequires/1.3.0@tools/master", "time": 1573724181.481263}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "tasking_tricore_installer/v6.2r2p2@tools/master", "time": 1573724181.5603507}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "ninja_installer/1.9.0@bincrafters/stable", "time": 1573724181.5704565}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "mydep1/2.1.2@tools/master", "time": 1573724181.5906913}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "mydep2/1.1.0@tools/master", "time": 1573724181.872878}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "python_scripts/1.0.0@tools/master", "time": 1573724181.8888807}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "mydep3/1.0.0@tools/master_stable", "time": 1573724181.955889}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "mydep4/5.5.4+build631@tools/master", "time": 1573724181.9679813}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "mydep3/1.0.0@tools/master_stable", "time": 1573724182.0298975}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "stubs/1.0.0@tools/master", "time": 1573724182.0465553}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "mydep3/1.0.0@tools/master_stable", "time": 1573724182.1135988}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "mydep5/1.1.0@tools/master_testing", "time": 1573724182.1302552}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "mydep6/3.8.7@tools/master_testing", "time": 1573724182.2039223}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "cmake_installer/3.15.3@conan/stable", "time": 1573724182.237956}
{"_action": "GOT_PACKAGE_FROM_LOCAL_CACHE", "_id": "mydep1/2.1.2@tools/master:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9", "time": 1573724182.3302727}
{"_action": "GOT_PACKAGE_FROM_LOCAL_CACHE", "_id": "mydep6/3.8.7@tools/master_testing:60f56ec37c39c591a5110964f671d57a04cb36de", "time": 1573724182.346931}
{"_action": "GOT_PACKAGE_FROM_LOCAL_CACHE", "_id": "cmake_installer/3.15.3@conan/stable:60b04a80ac4dd5c23f1acffacc679e9902e636e5", "time": 1573724182.346931}
{"_action": "GOT_PACKAGE_FROM_LOCAL_CACHE", "_id": "mydep2/1.1.0@tools/master:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9", "time": 1573724182.3635871}
{"_action": "GOT_PACKAGE_FROM_LOCAL_CACHE", "_id": "mydep4/5.5.4+build631@tools/master:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9", "time": 1573724182.3635871}
{"_action": "GOT_PACKAGE_FROM_LOCAL_CACHE", "_id": "ninja_installer/1.9.0@bincrafters/stable:456f15897172eef340fcbac8a70811f2beb26a93", "time": 1573724182.3802443}
{"_action": "GOT_PACKAGE_FROM_LOCAL_CACHE", "_id": "python_scripts/1.0.0@tools/master:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9", "time": 1573724182.3802443}
{"_action": "GOT_PACKAGE_FROM_LOCAL_CACHE", "_id": "stubs/1.0.0@tools/master:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9", "time": 1573724182.3969176}
{"_action": "GOT_PACKAGE_FROM_LOCAL_CACHE", "_id": "tasking_tricore_installer/v6.2r2p2@tools/master:305e69ff6241f1a61b2ece8034c51e0bc0d8c69d", "time": 1573724182.4039485}
{"_action": "GOT_PACKAGE_FROM_LOCAL_CACHE", "_id": "mydep5/1.1.0@tools/master_testing:e2695b6744fd69f13c4904b329803800fc708df2", "time": 1573724182.4039485}
{"_action": "COMMAND", "name": "build", "parameters": {"build_folder": null, "conanfile_path": "..", "install_folder": null, "package_folder": null, "should_build": false, "should_configure": true, "should_install": false, "should_test": false, "source_folder": null}, "time": 1573724192.892925}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "PyRequires/1.3.0@tools/master", "time": 1573724192.94152}
{"_action": "COMMAND", "name": "package", "parameters": {"build_folder": null, "install_folder": null, "package_folder": "Package", "path": "..", "source_folder": null}, "time": 1573724477.6134765}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "PyRequires/1.3.0@tools/master", "time": 1573724477.6624792}
{"_action": "COMMAND", "name": "export_pkg", "parameters": {"build_folder": null, "channel": "test", "conanfile_path": "..", "env": null, "force": true, "ignore_dirty": true, "install_folder": null, "lockfile": "conan.lock", "name": null, "options": null, "package_folder": "Package", "profile_names": null, "settings": null, "source_folder": null, "user": "gjess", "version": null}, "time": 1573724513.0307186}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "PyRequires/1.3.0@tools/master", "time": 1573724513.1427245}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "PyRequires/1.3.0@tools/master", "time": 1573724513.1597316}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "MyModule/1.1.1@gjess/test", "time": 1573724517.2659576}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "tasking_tricore_installer/v6.2r2p2@tools/master", "time": 1573724517.2779589}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "ninja_installer/1.9.0@bincrafters/stable", "time": 1573724517.2909606}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "mydep1/2.1.2@tools/master", "time": 1573724517.303964}
{"_action": "COMMAND", "name": "upload", "parameters": {"all_packages": true, "confirm": false, "integrity_check": false, "package": null, "pattern": "MyModule/1.1.1@gjess/test", "policy": null, "query": null, "remote_name": "my-tools-repo", "retry": null, "retry_wait": null}, "time": 1573724569.186804}
{"_action": "GOT_RECIPE_FROM_LOCAL_CACHE", "_id": "PyRequires/1.3.0@tools/master", "time": 1573724569.2578058}
{"_action": "REST_API_CALL", "duration": 0.08800673484802246, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Client-Anonymous-Id": "**********", "X-Client-Id": "gjess"}, "method": "GET", "time": 1573724569.3928142, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v1/ping"}
{"_action": "REST_API_CALL", "duration": 0.010998010635375977, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Client-Anonymous-Id": "**********"}, "method": "GET", "time": 1573724569.405813, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/files/conanmanifest.txt"}
{"_action": "REST_API_CALL", "duration": 0.04800271987915039, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Client-Anonymous-Id": "**********", "X-Client-Id": "gjess"}, "method": "GET", "time": 1573724569.4608166, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/users/check_credentials"}
{"_action": "REST_API_CALL", "duration": 0.2780177593231201, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Client-Anonymous-Id": "**********", "X-Client-Id": "gjess"}, "method": "GET", "time": 1573724569.7428353, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/files"}
{"_action": "REST_API_CALL", "duration": 0.012999296188354492, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Checksum-Deploy": "true", "X-Checksum-Sha1": "a1cc81db71ec9f5876badff8a6bf20891b372875", "X-Client-Anonymous-Id": "**********", "artifact_property_build.name": "JENKINS_JOB_1", "artifact_property_build.number": "12"}, "method": "PUT", "time": 1573724569.7598345, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/files/conanfile.py"}
{"_action": "REST_API_CALL", "duration": 0.0935220718383789, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Checksum-Sha1": "a1cc81db71ec9f5876badff8a6bf20891b372875", "X-Client-Anonymous-Id": "**********", "artifact_property_build.name": "JENKINS_JOB_1", "artifact_property_build.number": "12"}, "method": "PUT", "time": 1573724569.8573575, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/files/conanfile.py"}
{"_action": "REST_API_CALL", "duration": 0.011039495468139648, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Checksum-Deploy": "true", "X-Checksum-Sha1": "d53e15b32f58555b63b0dd847afd43d0f4776cdf", "X-Client-Anonymous-Id": "**********", "artifact_property_build.name": "JENKINS_JOB_1", "artifact_property_build.number": "12"}, "method": "PUT", "time": 1573724569.8713992, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/files/conanmanifest.txt"}
{"_action": "REST_API_CALL", "duration": 0.0280001163482666, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Checksum-Sha1": "d53e15b32f58555b63b0dd847afd43d0f4776cdf", "X-Client-Anonymous-Id": "**********", "artifact_property_build.name": "JENKINS_JOB_1", "artifact_property_build.number": "12"}, "method": "PUT", "time": 1573724569.9033606, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/files/conanmanifest.txt"}
{"_action": "UPLOADED_RECIPE", "_id": "MyModule/1.1.1@gjess/test", "duration": 0.6175539493560791, "files": [{"md5": "a0ab2f897f6e065774bb20d3e54048b1", "name": "conanfile.py", "path": "C:\\Users\\GJESS\\.conan\\data\\MyModule\\1.1.1\\gjess\\test\\export\\conanfile.py", "sha1": "a1cc81db71ec9f5876badff8a6bf20891b372875"}, {"md5": "af6052a7a48fd1555de4c7c546b67c42", "name": "conanmanifest.txt", "path": "C:\\Users\\GJESS\\.conan\\data\\MyModule\\1.1.1\\gjess\\test\\export\\conanmanifest.txt", "sha1": "d53e15b32f58555b63b0dd847afd43d0f4776cdf"}], "remote": "my-tools-repo", "time": 1573724569.9143608}
{"_action": "ZIP", "dst": "C:\\Users\\GJESS\\.conan\\data\\MyModule\\1.1.1\\gjess\\test\\package\\5b251bc59ed095ac3bf90b4ba1d82661dc846b37\\conan_package.tgz", "duration": 0.1350095272064209, "src": [{<zip contents>}], "time": 1573724570.1323748}
{"_action": "REST_API_CALL", "duration": 0.009000301361083984, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Client-Anonymous-Id": "**********", "X-Client-Id": "gjess"}, "method": "GET", "time": 1573724570.160376, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/users/check_credentials"}
{"_action": "REST_API_CALL", "duration": 0.01599884033203125, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Client-Anonymous-Id": "**********", "X-Client-Id": "gjess"}, "method": "GET", "time": 1573724570.1803763, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/packages/5b251bc59ed095ac3bf90b4ba1d82661dc846b37/revisions/7982b81a40ca8ca188423a30fc05343f/files"}
{"_action": "REST_API_CALL", "duration": 0.011002302169799805, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Checksum-Deploy": "true", "X-Checksum-Sha1": "34d57d5fe203eeff4fbe38d5ddd19e7bc0876f43", "X-Client-Anonymous-Id": "**********", "artifact_property_build.name": "JENKINS_JOB_1", "artifact_property_build.number": "12"}, "method": "PUT", "time": 1573724570.2003794, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/packages/5b251bc59ed095ac3bf90b4ba1d82661dc846b37/revisions/7982b81a40ca8ca188423a30fc05343f/files/conan_package.tgz"}
{"_action": "REST_API_CALL", "duration": 0.1670076847076416, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Checksum-Sha1": "34d57d5fe203eeff4fbe38d5ddd19e7bc0876f43", "X-Client-Anonymous-Id": "**********", "artifact_property_build.name": "JENKINS_JOB_1", "artifact_property_build.number": "12"}, "method": "PUT", "time": 1573724570.3713853, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/packages/5b251bc59ed095ac3bf90b4ba1d82661dc846b37/revisions/7982b81a40ca8ca188423a30fc05343f/files/conan_package.tgz"}
{"_action": "REST_API_CALL", "duration": 0.012997865676879883, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Checksum-Deploy": "true", "X-Checksum-Sha1": "24a1eb772f57d0107dcc54d446c7b83fc4927787", "X-Client-Anonymous-Id": "**********", "artifact_property_build.name": "JENKINS_JOB_1", "artifact_property_build.number": "12"}, "method": "PUT", "time": 1573724570.3873894, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/packages/5b251bc59ed095ac3bf90b4ba1d82661dc846b37/revisions/7982b81a40ca8ca188423a30fc05343f/files/conaninfo.txt"}
{"_action": "REST_API_CALL", "duration": 0.10600519180297852, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Checksum-Sha1": "24a1eb772f57d0107dcc54d446c7b83fc4927787", "X-Client-Anonymous-Id": "**********", "artifact_property_build.name": "JENKINS_JOB_1", "artifact_property_build.number": "12"}, "method": "PUT", "time": 1573724570.4963937, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/packages/5b251bc59ed095ac3bf90b4ba1d82661dc846b37/revisions/7982b81a40ca8ca188423a30fc05343f/files/conaninfo.txt"}
{"_action": "REST_API_CALL", "duration": 0.014003515243530273, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Checksum-Deploy": "true", "X-Checksum-Sha1": "1ba447f4e91dc6788e2c8e0990494cb812f1a170", "X-Client-Anonymous-Id": "**********", "artifact_property_build.name": "JENKINS_JOB_1", "artifact_property_build.number": "12"}, "method": "PUT", "time": 1573724570.5133975, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/packages/5b251bc59ed095ac3bf90b4ba1d82661dc846b37/revisions/7982b81a40ca8ca188423a30fc05343f/files/conanmanifest.txt"}
{"_action": "REST_API_CALL", "duration": 0.026000022888183594, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.20.3 (Python 3.7.3) python-requests/2.21.0", "X-Checksum-Sha1": "1ba447f4e91dc6788e2c8e0990494cb812f1a170", "X-Client-Anonymous-Id": "**********", "artifact_property_build.name": "JENKINS_JOB_1", "artifact_property_build.number": "12"}, "method": "PUT", "time": 1573724570.544399, "url": "https://myserver.artifactory.cm.mycompany.biz/artifactory/api/conan/my-tools-repo/v2/conans/MyModule/1.1.1/gjess/test/revisions/453a45120e2c2560fca63625364db2cd0be6601a/packages/5b251bc59ed095ac3bf90b4ba1d82661dc846b37/revisions/7982b81a40ca8ca188423a30fc05343f/files/conanmanifest.txt"}
{"_action": "UPLOADED_PACKAGE", "_id": "MyModule/1.1.1@gjess/test:5b251bc59ed095ac3bf90b4ba1d82661dc846b37", "duration": 0.6220376491546631, "files": [{"md5": "44678d013971e2fca16a1a1a29b942ef", "name": "conan_package.tgz", "path": "C:\\Users\\GJESS\\.conan\\data\\MyModule\\1.1.1\\gjess\\test\\package\\5b251bc59ed095ac3bf90b4ba1d82661dc846b37\\conan_package.tgz", "sha1": "34d57d5fe203eeff4fbe38d5ddd19e7bc0876f43"}, {"md5": "0653dad2640831e19de9e55453f9c991", "name": "conaninfo.txt", "path": "C:\\Users\\GJESS\\.conan\\data\\MyModule\\1.1.1\\gjess\\test\\package\\5b251bc59ed095ac3bf90b4ba1d82661dc846b37\\conaninfo.txt", "sha1": "24a1eb772f57d0107dcc54d446c7b83fc4927787"}, {"md5": "6d70e8e126540ca1665e2099978fb847", "name": "conanmanifest.txt", "path": "C:\\Users\\GJESS\\.conan\\data\\MyModule\\1.1.1\\gjess\\test\\package\\5b251bc59ed095ac3bf90b4ba1d82661dc846b37\\conanmanifest.txt", "sha1": "1ba447f4e91dc6788e2c8e0990494cb812f1a170"}], "remote": "my-tools-repo", "time": 1573724570.5584006}
```

**Comment by GordonJess** [comment]
Hi @czoido, I noticed that [issue 6080](https://github.com/conan-io/conan/issues/6080) was created to solve this issue when `scm` is used to get sources.  So I tried to workaround this issue by removing the `scm` attribute, instead using `exports_sources`.  I can now see `conan_sources.tgz` in artifactory but I am still receiving the same error.  Just wanted to report in case it helps.  Looking forward to this!
